{
  if (this.cause instanceof Exception) {
    return (Exception)cause;
  }
  return super.getException();
}
